libctru/include/3ds/thread.h

```diff
@@ -31,6 +31,9 @@ typedef struct Thread_tag* Thread;
  * - Processor #2 is New3DS exclusive. Normal applications can create threads on this core if the exheader kernel flags bitmask has 0x2000 set.
  * - Processor #3 is New3DS exclusive. Normal applications cannot create threads on this core.
  * - Processes in the BASE memory region can always create threads on processors #2 and #3.
+ * 
+ * @note For the detached parameter to work, you need to either return from your entrypoint or call threadExit. 
+ *       svcExitThread should not be used, or you will have to call threadFree yourself.
```

Comment:
This note can be summed up with: `svcExitThread should never be used inside the thread; use threadExit instead.`. No need to talk about an implementation detail not working if you don't do the right thing.

Also you can't free yourself using `threadFree`.